tests/user.test.js

```
@@ -25,7 +25,7 @@ describe('User', () => {
     try {
       await sequelize.sync({ force: true });
     } catch (err) {
-      console.log(err);
+      // Error occurred
```

Avoid silently swallowing errors in beforeAll. The `catch` block at this line is empty which will hide failures from `sequelize.sync({ force: true })`. Per the task, model sync must succeed when running tests, so surface or log this error to make debugging easier (e.g. `console.error(err)` or rethrow).